docs/conf/vilnius/2019/news/announcing-talks.rst

```diff
@@ -0,0 +1,57 @@
+:template: {{year}}/generic.html
+
+.. post:: May 16, 2019
+   :tags: vilnius-2019, speakers, tickets, visiting
+
+Announcing Speakers and Talks
+=============================
+
+Greetings, fellow documentarians! We're writing today with an update on the fast-approaching Vilnius event, and we're very excited to announce the speakers and talks for the event.
+
+Thank you so much to everyone who took the time to send us a pitch – we literally wouldn't have an event without you!
+
+We also have some info about ticket purchasing (now's the time!), trip planning (Vilnius is beautiful!), and Writing Day (start thinking about projects!).
+
+Read on for more details!
+
+Full speaker line-up
+--------------------
+
+Our events bring a wide range of voices to the Write the Docs stage. Because the role of "documentarian" looks so different to each of us, we aim for a line-up that spans a good mix of practical, philosophical, and technical topics – with the odd whimsical one thrown in for good measure.
+
+We're really excited about the presentations we've got for this first-year event, and we hope you are too!
+
+.. datatemplate::
+   :source: /_data/{{year}}.{{shortcode}}.speakers.yaml
+   :template: {{year}}/speakers-simple-list.rst
+   :include_context:
+
+Head over to the `Speakers page <https://www.writethedocs.org/conf/{{shortcode}}/{{year}}/speakers/>`_ to see the full abstracts. We hope there’s something there for everyone to enjoy, and also something that will broaden your horizons a bit.
+
+Tickets are going fast!
+-----------------------
+
+If you don't have your ticket yet, now's a great time to snap one up. Conference tickets have already been going quickly, and we have a limited capacity, so if you're considering attending get off the fence and register now!
+
+`Buy your tickets here! <https://www.writethedocs.org/conf/vilnius/2019/tickets/>`_
+
+Plan your trip
+--------------
+
+In the final weeks leading up to the event, we’ll be announcing more details about the conference events, including the full schedule, writing day, and social event on Sunday.
+
+We invite you to check out the `Visit Vilnius <https://www.writethedocs.org/conf/vilnius/2019/visiting/>`_ page on the conference website, and the local team is happy to answer your questions in the `#wtd-conferences <https://writethedocs.slack.com/messages/wtd-conferences>`_ channel on Slack.
```

Comment:
When are we sending this out? Because #wtd-conferences may be filled with PDX stuff and Vilnius questions may very easily be overlooked in all the other buzz.

Reply:
I'm not worried about that, the amount of Vilnius questions is unlikely to be high (we had 2 in the last month) and the locals are monitoring the channel. We're sending this post out today.